src/firefly/js/templates/lightcurve/LcPhaseFoldingPopup.jsx

```diff
@@ -0,0 +1,623 @@
+/*
```

Comment:
I think it would be better to break out some of the non-UI function into a separate file.  A large percent of this file is not UI.  It will be easier to write test.

We are not always consistent about this but I think it is better to try to keep mostly only jsx in a jsx file.